combined_robot_hw_tests/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(combined_robot_hw_tests)
```

Comment:
It is the same approach we have for `controller_manager_tests` already btw

On 21 May 2016 at 17:37, Dave Coleman <notifications@github.com> wrote:

> In combined_robot_hw_tests/CMakeLists.txt
> <https://github.com/ros-controls/ros_control/pull/231#discussion_r64136939>
> :
>
> > @@ -0,0 +1,63 @@
> > +cmake_minimum_required(VERSION 2.8.3)
> > +project(combined_robot_hw_tests)
>
> I am still wavering on this, but I see your point about hiding the test
> plugins and agree with your choice
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ros-controls/ros_control/pull/231/files/2c7c55e7f357372a2421e00104ec36c7347b5206#r64136939>
>